tests/test_exporter.py

```
@@ -399,3 +399,54 @@ def test_extract_media_concatenation(test_db):
+        def mock_exists_fallback(path):
+            """Mock os.path.exists so it falls back to .en.mkv and finds it."""
+            if path == "/fake/path/Belle (2021).en.mkv":
+                return True
+            return False
+
+        with patch("os.path.exists", side_effect=mock_exists_fallback):
+            success, msg, _, _, _ = exporter.extract_media(sid, "/fake/out")
+            assert success is True
+
+        # Now test when both exist, stripped is preferred
+        def mock_exists_both(path):
```

Ruff (0.16.2)

[warning] Missing return type annotation for private function `mock_exists_both`

Add return type annotation: `bool`

(ANN202)